build/configs/artik053/README.md

```diff
@@ -93,11 +93,18 @@ openocd -f artik053.cfg -c ' \
 
 ## ROMFS
 
+### Pre-condition
```

Comment:
No, not here. genromfs is a common tool. There is no dependancy with board. I gave a link to tools/fs/README_ROMFS.md